WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -95,4 +128,35 @@
   });
 
   $('#display-<%= @display %>').button('toggle');
+
+  // From http://stackoverflow.com/questions/901115/how-can-i-get-query-string-values-in-javascript/901144#901144
+  function getParameterByName(name, url) {
```

Comment:
I don't quiet follow but just in case we still need this, I think we already have a method like this, see the code [here](https://github.com/cubing/worldcubeassociation.org/blob/master/WcaOnRails/app/assets/javascripts/jquery_plugins.js#L12-L18) and an example usage [here](https://github.com/cubing/worldcubeassociation.org/blob/4fed8ead939cb979a38fb76b69a560b942a872a0/WcaOnRails/app/assets/javascripts/persons.js#L18).